src/bin/cargo/commands/owner.rs

```diff
@@ -6,25 +6,68 @@ use cargo::util::auth::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg_quiet()
         .arg(Arg::new("crate").action(ArgAction::Set))
-        .arg(
-            multi_opt(
-                "add",
-                "LOGIN",
-                "Name of a user or team to invite as an owner",
-            )
-            .short('a'),
-        )
-        .arg(
-            multi_opt(
-                "remove",
-                "LOGIN",
-                "Name of a user or team to remove as an owner",
-            )
-            .short('r'),
+        .arg_required_else_help(true)
+        .override_usage(
+            "\
+       cargo owner [OPTIONS] add    OWNER_NAME <CRATE_NAME>
```

Comment:
Side note: clap-rs/clap#1334 would be helpful for this.  It would extend clap's subcommand help to allow cases like `git stash` where the subsubcommands (e.g. `git stash push`) are extensions of the subcommand (`git stash`)